switchbot_ros/src/switchbot_ros/switchbot.py

```diff
@@ -21,6 +32,29 @@ def __init__(self, token):
         self.update_device_list()
         self.update_scene_list()
 
+    def make_sign(self, token, secret):
+        
+        nonce = uuid.uuid4()
+        t = int(round(time.time() * 1000))
+        string_to_sign = '{}{}{}'.format(token, t, nonce)
+        
+        string_to_sign = bytes(string_to_sign, 'utf-8')
+        secret = bytes(secret, 'utf-8')
+        
+        sign = base64.b64encode(hmac.new(secret, msg=string_to_sign, digestmod=hashlib.sha256).digest())
+                
+        return sign, str(t), nonce
+
+    def make_request_header(self, token, secret):
+        sign,t,nonce = self.make_sign(token, secret)
```

Comment:
```python
sign, t, nonce = self.make_sign(token, secret)
```

Reply:
I inserted the spaces.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491/commits/dc12bac96f632ccd1cc78f94ab69c8ae24475cf2#diff-396497c868baf8630a5e57ccf98059c7d6f1d45e91faccfe0917e1012f124191R55